controllers/authpolicy_controller.go

```diff
@@ -157,10 +156,6 @@ func (r *AuthPolicyReconciler) reconcileResources(ctx context.Context, ap *api.A
 		return err
 	}
 
-	if err := r.reconcileIstioAuthorizationPolicies(ctx, ap, targetNetworkObject, gatewayDiffObj); err != nil {
```

Comment:
This will cause the AuthPolicy `Enforced` status condition to no longer be tied to the status of the internal resource that implements the call to ext authz, Istio AuthorizationPolicy in this case. We need some other way to do that now with the concurrent reconciliation of this resource, and the same for Envoy SecurityPolicies.

Reply:
There is one specific controller for the istio AuthorizationPolicy and one specific controller for the EnvoyGateway SecPolicy resources. Current functionality for Istio should remain exactly the same (if not improved).